spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_service.py

```diff
@@ -169,8 +169,7 @@ def __init__(self, stub_factory, options):
     self.__stackdriver_options.update(options.get('stackdriver', {}))
     self.__stub_factory = stub_factory
     self.__stub = None
-    self.__project = options.get('project',
-                                 self.__stackdriver_options.get('project'))
+    self.__project = self.__stackdriver_options.get('project')
```

Comment:
I think this should be "options.get('project') or self.__stackdriver_options.get('project')" so that you could still override it with the commandline. The command line is a useful for some admin commands.

Reply:
I know the functionally of dictionary get() method, so I thought as you it will retun the project id from cli or yaml file. However I found project id is always `''` empty string, and get() method returns it because 'project' variable exists even if it is not a valid value.
And I found the code to override project id from cli already. Do you need a reason to override twice?

```
self.__stackdriver_options = dict(options.get('stackdriver', {}))
    for key in ['project', 'zone', 'instance_id', 'credentials_path']:
      if options.get(key):
        self.__stackdriver_options[key] = options[key]  # commandline override
```